gw_eccentricity/gw_eccentricity.py

```diff
@@ -63,29 +63,52 @@ def measure_eccentricity(tref_in=None,
                          method="Amplitude",
                          dataDict=None,
                          num_orbits_to_exclude_before_merger=2,
+                         precessing=False,
                          extra_kwargs=None):
     """Measure eccentricity and mean anomaly from a gravitational waveform.
 
-    Eccentricity is measured using the GW frequency omega22(t) =
-    dphi22(t)/dt, where phi22(t) is the phase of the (2, 2) waveform
-    mode. We currently only allow time-domain, nonprecessing waveforms. We
-    evaluate omega22(t) at pericenter times, t_pericenters, and build a
-    spline interpolant omega22_pericenters(t) using those data
-    points. Similarly, we build omega22_apocenters(t) using omega22(t) at
-    the apocenter times, t_apocenters.
+    Eccentricity is measured using the GW frequency omega_gw(t) =
+    d(phase_gw)/dt, where phase_gw(t) is the phase of the (2, 2) waveform mode
+    for nonprecessing systems. For precessing systems, phase_gw is obtained
+    using an antisymmetric combination of (2, 2) and (2, -2) mode phases in the
+    coprecessing frame.
 
```

Comment:
Get rid of " =
    d(phase_gw)/dt, where phase_gw(t) is the phase of the (2, 2) waveform mode
    for nonprecessing systems. For precessing systems, phase_gw is obtained
    using an antisymmetric combination of (2, 2) and (2, -2) mode phases in the
    coprecessing frame."

You repeat that bit below anyway. Just merge the next paragraph with this after deleting that bit.

That is:
Eccentricity is measured using the GW frequency omega_gw(t) = d(phase_gw)/dt. Throughout this...

Reply:
Done